.travis.yml

```diff
@@ -15,6 +15,7 @@ services:
 addons:
   apt:
     packages:
+      # TODO upgrade to protobuf 3.5 or higher
```

Comment:
todo complete as of below using protobuf 3.11.4?

Reply:
removed